functions.php

```diff
@@ -14,11 +14,65 @@ function getData($db)
     return $plants;
 }
 
-function processData($plants)
-{
+function getHeadings($plants) {
+    $plantHeading = '<tr>';
+    foreach ($plants[0] as $key => $value) {
+        $plantHeading .= '<th>' . $key . '</th>';
+    }
+    $plantHeading .= '</tr>';
+    return $plantHeading;
+}
+
+function getTableBody($plants) {
+    $tableBody = '';
+    foreach ($plants as $plant) {
+        $tableBody .= '<tr>';
+        $tableBody .=
+            '<td>' . $plant['id'] . '</td>' .
+            '<td>' . $plant['common_name'] . '</td>' .
+            '<td>' . $plant['latin_name'] . '</td>' .
+            '<td>' . $plant['identification_image'] . '</td>' .
+            '<td>' . $plant['canopy_level'] . '</td>' .
+            '<td>' . $plant['life_span'] . '</td>';
+        $tableBody .= '</tr>';
+    }
+    return $tableBody;
```

Comment:
You are likely to find using <divs> laid out with CSS more flexible than <table> elements when you get to styling your page. <table> comes with a lot of 'table stuff' that can be harder to work with.